.github/workflows/verify-bundled-files.yml

```diff
@@ -0,0 +1,43 @@
+name: Verify Bundled Files
+
+on:
+  push:
+  pull_request:
+  schedule:
+   - cron: "0 1 * * *"
+  workflow_dispatch: ~
+
+permissions:
+  contents: read
+
+jobs:
+  VERIFY_BUNDLED_FILES:
+    name: Verify Bundled Files
+    runs-on: ubuntu-22.04
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v5
+
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          base: master
+          filters: |
+            opcache_jit_ir:
+              - 'ext/opcache/jit/ir/**'
+
+      - name: Opcache JIT IR - Download
+        if: ${{ !cancelled() && (steps.changes.outputs.opcache_jit_ir == 'true' || (github.event_name != 'push' && github.event_name != 'pull_request')) }}
+        uses: actions/checkout@v5
+        with:
+          repository: dstogov/ir
+          ref: 5a81104e650ebd7ac24eb63d4dff67db723a5278
```

Comment:
Do you propose to change this file every time when IR is updated?
Is it possible to parse GIT commit? or introduce some special file in the IR directory instead?